internal/pkg/cli/storage_init.go

```diff
@@ -21,30 +22,37 @@ import (
 )
 
 const (
-	dynamoDBStorageType = "DynamoDB"
-	s3StorageType       = "S3"
-)
-
-const (
-	s3BucketFriendlyText      = "S3 Bucket"
-	dynamoDBTableFriendlyText = "DynamoDB Table"
-)
-
-const (
-	ddbKeyString = "key"
+	dynamoDBStorageType     = "DynamoDB"
+	s3StorageType           = "S3"
+	rdsStorageType          = "RDS"
 )
 
 var storageTypes = []string{
 	dynamoDBStorageType,
 	s3StorageType,
+	//rdsStorageType, // Hide RDS option for now.
```

Comment:
Instead of dead code, can we remove this line and just add it later when we need it?